tests/issues/issue-2475.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+return [
+    'issue-2370-A' => [
+        'ua' => 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4240.75 Safari/537.36',
+        'properties' => [
+            'Comment' => 'Chrome 99.0',
+            'Browser' => 'Chrome',
+            'Browser_Type' => 'Browser',
+            'Browser_Bits' => '64',
+            'Browser_Maker' => 'Google Inc',
+            'Browser_Modus' => 'unknown',
+            'Version' => '99.0',
+            'Platform' => 'Win10',
+            'Platform_Version' => '10.0',
+            'Platform_Description' => 'Windows 10',
+            'Platform_Bits' => '64',
+            'Platform_Maker' => 'Microsoft Corporation',
+            'Alpha' => false,
+            'Beta' => false,
+            'Frames' => true,
+            'IFrames' => true,
+            'Tables' => true,
+            'Cookies' => true,
+            'JavaScript' => true,
+            'VBScript' => false,
+            'JavaApplets' => false,
+            'isSyndicationReader' => false,
+            'isFake' => false,
+            'isAnonymized' => false,
+            'isModified' => false,
+            'CssVersion' => '3',
+            'Device_Name' => 'Windows Desktop',
+            'Device_Maker' => 'unknown',
+            'Device_Type' => 'Desktop',
+            'Device_Pointing_Method' => 'mouse',
+            'Device_Code_Name' => 'Windows Desktop',
+            'Device_Brand_Name' => 'unknown',
+            'RenderingEngine_Name' => 'Blink',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'Google Inc',
+        ],
+        'lite' => false,
+        'standard' => true,
+        'full' => true,
+    ],
+    'issue-2370-B' => [
```

Comment:
Please rename this test to "issue-2475-B"

Reply:
done